src/main/java/com/exasol/dbbuilder/dialects/mysql/MySqlTable.java

```diff
@@ -0,0 +1,85 @@
+package com.exasol.dbbuilder.dialects.mysql;
+
+import com.exasol.db.Identifier;
+import com.exasol.dbbuilder.dialects.*;
+
+/**
+ * A MySql table that allows specifying a charset.
+ */
+public class MySqlTable extends Table {
+
+    private final String charset;
+
+    /**
+     * Create a new MySql table based on a given builder.
+     *
+     * @param builder the builder from which to copy the values
+     */
+    protected MySqlTable(final Builder builder) {
+        super(builder);
+        this.charset = builder.charset;
+    }
+
+    /**
+     * Get the table's charset.
+     * 
+     * @return charset or {@code null} for the default charset
+     */
+    public String getCharset() {
+        return charset;
+    }
+
+    /**
+     * Create a builder for a {@link MySqlTable}.
+     *
+     * @param writer       database object writer
+     * @param parentSchema parent schema
+     * @param tableName    name of the database table
+     * @return new {@link Builder} instance
+     */
+    public static Builder builder(final DatabaseObjectWriter writer, final Schema parentSchema,
+            final Identifier tableName) {
+        return new Builder(writer, parentSchema, tableName);
+    }
+
+    /**
+     * Builder for {@link MySqlTable}s.
+     */
+    public static class Builder extends Table.Builder {
+
+        private String charset;
+
+        private Builder(final DatabaseObjectWriter writer, final Schema parentSchema, final Identifier tableName) {
+            super(writer, parentSchema, tableName);
+        }
+
+        @Override
```

Comment:
Why override the base function if it is unmodified? For the `this` to be the parent type?

Reply:
Yes, exactly. I added a comment to clarify this.